Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "near"
 version = "2.0.0"
```

Comment:
Version should be bumped here.

Reply:
2.0.0 has not been released/published yet, I don't see the reason to bump it.

Reply:
We have already deployed it on our test provider, so you need to bump it, sorry.

Reply:
@agrojean-ledger Do you suggest we bump it to 2.1.0?

Reply:
Yes, just bump the minor version :ok:

Reply:
Done